app/templates/eventNav.html

```diff
@@ -4,11 +4,19 @@
 {% block app_content %}
     {% block content_pageHeading %}
     <div class="text-center">
-        <h1 id="title">{{page_title}}</h1>
-        {% if eventData.name != 'All Volunteer Training' and eventData['program'] == None %}
-            <h3>CELTS-Sponsored Event</h3>
-        {% elif eventData['program'] %}
-            <h3>{{eventData["program"].programName}} Event</h3>
+        {% if g.current_user.isAdmin %}
+            <h1 id="title">{{page_title}}</h1>
+            {% if eventData.name != 'All Volunteer Training' and eventData['program'] == None %}
+                <h3>CELTS-Sponsored Event</h3>
+            {% elif eventData['program'] %}
+                <h3>{{eventData["program"].programName}} Event</h3>
```

Comment:
Lets add "A" or "An" depending on the next word at the start so it says "A `programName` Event".